wikimedia/wikipedia-ios
# T352881: Add Bold & Italics shortcuts to Native Editor

**Phabricator: [T352881](https://phabricator.wikimedia.org/T352881)**

### Notes
* Tested only on Simulator with my Mac's keyboard

### Test Steps on Simulator
1.  Go to editor for any page
2. Make sure hardware keyboard is connected (I/O -> Keyboard -> Connect hardware keyboard)
3. Select some text
4. Press CMD+B or CMD+I
